library/core/src/slice/ascii.rs

```diff
@@ -55,6 +59,96 @@ impl [u8] {
             byte.make_ascii_lowercase();
         }
     }
+
+    /// Slice version of [`u8::escape_ascii`].
```

Comment:
I think it's clearer to say something along the lines of:

```suggestion
    /// Escape this byte slice as an ASCII sequence.
```